README.md

```diff
@@ -1,17 +1,242 @@
+- [MinimumApplicationLaunchCount](https://github.com/nventive/ReviewService/blob/a78e37c7e9b6fbe07ba1fec5d0f2b3b2f31bf356/src/ReviewService.Abstractions/ReviewConditionsBuilder.Extensions.cs#L49) : Make sure that it prompts for review only if the number of times the application has been launched meets the required minimum.
+- [MinimumTimeElapsedSinceApplicationFirstLaunch](https://github.com/nventive/ReviewService/blob/a78e37c7e9b6fbe07ba1fec5d0f2b3b2f31bf356/src/ReviewService.Abstractions/ReviewConditionsBuilder.Extensions.cs#L65) : Make sure that it prompts for review only if the time elapsed since the first application launch meets the required minimum.
+- [Custom](https://github.com/nventive/ReviewService/blob/1484f946c60e2bf1cb86b27faa60c148a1e56d45/src/ReviewService.Abstractions/ReviewConditionsBuilder.Extensions.cs#L83) : Custom condition made with a synchronous function lambda.
+- [CustomAsync](https://github.com/nventive/ReviewService/blob/1484f946c60e2bf1cb86b27faa60c148a1e56d45/src/ReviewService.Abstractions/ReviewConditionsBuilder.Extensions.cs#L97) : Custom asynchronous condition made with an asynchronous function lambda.
+
+Here is a simple code that uses the builder extensions for review conditions.
+
+ ``` cs
+var reviewConditionsBuilder = ReviewConditionsBuilder.Empty()
+    .MinimumPrimaryActionsCompleted(1)
+    .MinimumSecondaryActionsCompleted(1)
+    .MinimumApplicationLaunchCount(1)
+    .MinimumTimeElapsedSinceApplicationFirstLaunch(TimeSpan.FromDays(1));
+```
+
+### Default Tracking Extensions
+
+- [TrackApplicationLaunched](https://github.com/nventive/ReviewService/blob/a78e37c7e9b6fbe07ba1fec5d0f2b3b2f31bf356/src/ReviewService.Abstractions/ReviewService.Extensions.cs#L20) : Tracks that the application was launched (Also tracks if it's the first launch).
+- [TrackPrimaryActionCompleted](https://github.com/nventive/ReviewService/blob/a78e37c7e9b6fbe07ba1fec5d0f2b3b2f31bf356/src/ReviewService.Abstractions/ReviewService.Extensions.cs#L45) : Tracks that a primary action was completed.
+- [TrackSecondaryActionCompleted](https://github.com/nventive/ReviewService/blob/a78e37c7e9b6fbe07ba1fec5d0f2b3b2f31bf356/src/ReviewService.Abstractions/ReviewService.Extensions.cs#L61) : Tracks that a secondary action was completed.
+
+### Customized Review Settings
+
+If you need custom conditions for your application, you will have to create another record that inherits from `ReviewSettings`.
+
+ ``` cs
+using ReviewService.Abstractions;
+
+/// <summary>
+/// The custom review prompt settings used for prompt conditions.
+/// </summary>
+public record ReviewSettingsCustom : ReviewSettings
+{
+    /// <summary>
+    /// Gets or sets if the application onboarding has been completed.
+    /// </summary>
+    public bool HasCompletedOnboarding { get; init; }
+}
+```
+
+### Customized Review Conditions
```

Comment:
Shouldn't we also specify that you can use the `Custom` and `CustomAsync` extension method here?